[Example] Change widget app to ui app @open sesame 05/30 00:23

zhoonit opened the pull request:
**Changes proposed in this PR:**
- The scaffolding was widget app, but widget has one layout 
  limitation. This PR ports widget app to ui app.

**Minor changes proposed in this PR:**
- Add handler for 'routes/to' signal.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

jijoongmoon commented:
Recently, we updated CI Server. Due to the some changes of CI Server, you have to rebase with recent master. ( Recent update is mainly android related. )

zhoonit commented:
Recently, we updated CI Server. Due to the some changes of CI Server, you have to rebase with recent master. ( Recent update is mainly android related. )

> Recently, we updated CI Server. Due to the some changes of CI Server, you have to rebase with recent master. ( Recent update is mainly android related. )

Updated :)
